app/assets/javascripts/components/organization_display.jsx

```diff
@@ -1,39 +1,64 @@
 var OrganizationsDisplay = React.createClass({
+
   getInitialState: function() {
-    return { searchTerm: null, techSizeSearch: null, techStacks: null };
+    return { organizations: this.props.organizations, searchTerm: null, techSizeSearch: null, technologySearch: null };
   },
-  filterOrganizations: function() {
-    var searchTerm = this.refs.searchInput.value.toLowerCase();
-    this.setState({ searchTerm: searchTerm });
+
+  getTechValue: function(string){
+    var arr = string.split(",");
+    var outArr = arr.join("+");
+    this.setState({
+      technologySearch: outArr
+    }, this.filterOrganizations);
   },
-  filterOrganizationTechSize: function() {
+  
+  filterOrganizations: function() {
+    var termSearch = this.refs.searchInput.value.toLowerCase();
     var techSizeSearch = this.refs.sizeInput.value;
-    this.setState({ techSizeSearch: techSizeSearch });
+    // var technologySearch = "";
+    var url = "search/";
+    var that = this;
+    $.ajax({
+      url: url,
+      method: "get",
+      dataType: 'json',
+      data: {
+        term: termSearch,
+        size: techSizeSearch,
+        tech: this.state.technologySearch
+      },
```

Comment:
is this working properly for you? Generally speaking with `GET` requests you send params as query string data (part of the url)